app/views/account/admin/users/_user.html.slim

```diff
@@ -2,13 +2,13 @@ tr
   td= user.email
   td= user.full_name
   td
-    - if policy(user).toggle_ban?
+    - if admin_or_moderator? && user.reporter?
       - if user.banned?
         = link_to 'Розблокувати', toggle_ban_account_admin_user_path(user), method: :patch
       - else
         = link_to 'Заблокувати', toggle_ban_account_admin_user_path(user), method: :patch
   td
-    - if policy(user).change_role?
+    - if admin? && !user.admin?
```

Comment:
Current user - admin and user whom we want to change role - not admin. The idea is that admin role can not be changed by other admins